CBinding/Refactoring/RefactoryCommands.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace CBinding.Refactoring
+{
+	/// <summary>
+	/// Enumeration containing custom refactoring commands
+	/// </summary>
+	public enum RefactoryCommands
+	{
+		GotoDefinition,
```

Comment:
which one should i use? MonoDevelop.Refactoring.RefactoryCommands does not contain a GoToDefinition